include/openenclave/internal/eeid.h

```diff
@@ -28,6 +28,19 @@ static const uint8_t OE_DEBUG_PUBLIC_KEY[] = {
     0xce, 0x73, 0xe4, 0x33, 0x63, 0x83, 0x77, 0xf1, 0x79, 0xab, 0x44,
     0x56, 0xb2, 0xfe, 0x23, 0x71, 0x93, 0x19, 0x3a, 0x8d, 0xa};
 
+/** Struct to track EEID-relevant SGX claims. */
+typedef struct
+{
+    uint8_t* enclave_hash;
+    size_t enclave_hash_size;
+    uint8_t* signer_id;
+    size_t signer_id_size;
+    uint16_t product_id;
+    uint32_t security_version;
+    uint64_t attributes;
+    uint32_t id_version;
+} oe_eeid_relevant_sgx_claims_t;
```

Comment:
This header file is currently not in an sgx directory or having sgx in the filename, but this type has sgx in the name.  Can we keep sgx types in a file either with sgx in the name or under an sgx subdir?

Reply:
Fixed, the SGX in the name wasn't really necessary.